troubleshooting/devtools.md

```diff
@@ -0,0 +1,188 @@
+# Dev Tools Guide
+
+## VS Code Remote Setup
+**VSCode Remote**  
+After installing [VS Code](https://code.visualstudio.com/Download), the following tutorial can be used to connect Visual Studio Code to a remote Linux system or server. <https://code.visualstudio.com/docs/remote/ssh>
```

Comment:
I wonder prefer to add link to a string rather than show link directly. like [Remote Development using SSH](https://code.visualstudio.com/docs/remote/ssh)